content/basics/vimmodes/insert.md

```diff
@@ -4,4 +4,18 @@ date =  2017-10-30T10:48:58-04:00
 weight = 6
 +++
 
-Insert Mode
+Insert Mode is where you can write/edit your documments. There are some keys to enter in this mode, let's see them:  
+
+| Key     | Description   																											|
+|---------|---------------------------------------------------------------------|
+| **i**   | This key enter in insert mode before under the cursor								|
+| **I**   | This key enter in insert mode begin line under the cursor						|
+| **a**   | This key enter in insert mode after under the cursor 								|
+| **A**   | This key enter in insert mode end line under the cursor  						|
+| **o**   | This key enter in insert mode on the next line under the cursor  		|
+| **O**   | This key enter in insert mode on the previous line under the cursor |
```

Comment:
| **O**   | This key switches to insert mode adding a line above the current one  		|